app/views/provider_interface/reports/mid_cycle_reports/show.html.erb

```diff
@@ -0,0 +1,50 @@
+<%= content_for :title, 'Mid cycle report'%>
+<%= content_for :before_content, breadcrumbs(t('page_titles.provider.reports') => provider_interface_reports_path,
+                                             'Mid cycle report' => nil) %>
+
+                                           
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">
+      <span class="govuk-caption-l"> <%= @provider.name %></span>
+      2022 to 2023 recruitment cycle performance
+    </h1>
+      <p class="govuk-body">This report shows your organisation’s initial teacher training (ITT) recruitment performance from 11 October 2022 to 3 April 2023.</p>
+      <p class="govuk-body">The report compares your organisation’s recruitment data this cycle, to the same time last recruitment cycle. It also includes national level data.</p>
+      <h2 class="govuk-heading-m">How the data in this report compares to other data we publish</h2>
```

Comment:
Under this header, can we replace the content with:

Previously, we shared a monthly statistics page with training providers. On that page, we calculate data using the same method UCAS used to. This method counts a candidate’s furthest application. For example, if an application moves from ‘recruited’ to ‘deferred’ the recruited count will go down, and the deferred count will go up. This method does not easily show where a candidate ends up over time. 

This page about your recruitment performance, calculates data in a new way by counting all submitted applications, regardless of their outcome. It also counts all offers regardless of if the offer is accepted, declined or waiting for a decision.  

With each page using a different method, you should not compare this page with the monthly statistics page.

Reply:
🆗